pkg/common/config/config.go

```diff
@@ -107,6 +108,11 @@ var (
 	// ErrUsernameMissing is returned when the provided username is empty.
 	ErrUsernameMissing = errors.New("username is missing")
 
+	// ErrInvalidUsername is returned when vCenter username provided in vSphere
+	// config secret is invalid. e.g. If username doesn't contain domain name, then
+	// it will be considered as invalid username.
+	ErrInvalidUsername = errors.New("username is invalid, make sure that username contains domain name")
```

Comment:
`username is invalid, make sure that username contains domain name` ==> `username is invalid, make sure it is a fully qualified domain username`

Reply:
Updated the error message accordingly.